typescript/cli/src/config/ism.ts

```diff
@@ -111,6 +112,7 @@ export async function createIsmConfig(
 export async function createMultisigConfig(
   type: IsmType.MERKLE_ROOT_MULTISIG | IsmType.MESSAGE_ID_MULTISIG,
 ): Promise<MultisigIsmConfig> {
+  creating(type);
```

Comment:
could had a parent function that these are passed into to wrap with these logs around instead, or just remove them altogether. lmk what you think